decentraland/unity-renderer
# feat: biw refactor of the builder editor

## What does this PR change?

Fixes #1309 

This a refactor that increase the readability of the feature and split the functionality between different scripts

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=feat/biw-new-scene
2. Check that builder in world works as expected

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
